pom/PageRegister.spec.ts

```diff
@@ -0,0 +1,45 @@
+import{ Page } from "@playwright/test";
+
+export default class PageRegister{
+    constructor (public page: Page){
+
+    }
+
+    fieldFirstName: string = "//input[@id='input-firstname']"
```

Comment:
https://playwright.dev/docs/pom
please follow the structure of playwright doc

Reply:
Updated script follow the structure of playwright doc